bugsnag-android-core/src/main/java/com/bugsnag/android/ImmutableConfig.kt

```diff
@@ -30,7 +28,8 @@ internal data class ImmutableConfig(
     val maxBreadcrumbs: Int,
     val maxPersistedEvents: Int,
     val maxPersistedSessions: Int,
-    val persistenceDirectory: File
+    val persistenceDirectory: File,
+    val user: User
```

Comment:
Adding `user` to `ImmutableConfig` seems to break the pattern that the class was created for as the user can be changed in the client later, so is not one of the fields considered immutable configuration.

I can see it provides a useful mechanism to pass into the UserStore, but it seems like a bit of a compromise in the design to me. Can we pass a `User` into `UserStore.load` that represents the active user in the client?

Reply:
Updated to pass the initial user object in via `UserStore#load`.